### ISSUE TITLE
Request: Allow fixing namespace for whole project/solution

### ISSUE THREAD

**Issue by chvogel** [pauer24/VsNamespaceFixer#17]
**Request:**
Allow fixing namespace for whole project/solution with single action.

**Current workaround:**
For each project:
- Select all files in project root -> right-click -> Adjust namespace
- Select all folders in project root -> right-click -> Adjust namespace

**Reasoning:**
I mainly use this extension at irregular intervals (for example after big merges) to verify that the namespaces for all files in our solution (700 files in 15 projects) are correct.
Having a single action for a solution-wide check instead of having to select each project's folders/files would allow me to use this extension for every merge instead of just occasionally when the time allows.

**Comment by pauer24** [comment]
I completly agree. I tried twice, but both times ended without the way to do it.

Feel free to dig into this and send a PR!! :D

**Comment by Tr4ncer** [comment]
Yes, it is achievable. If I had to do it, I would use the information contained in msbuild.

**Comment by pauer24** [comment]
Yes, I know as well it is achievable, but for the usage I'd give to this improvement and the time it would take to implement it, it's not worth it for me. But be my guest! And please, implement it yourself! I have already reviewed, accepted and uploaded new versions from PR all around.
Cheers!

**Comment by Tr4ncer** [comment]
Ok, I may be watching this now.
Finally, if @pauer24 still agrees.

**Currently 'Adjust namespaces' is available:**
* on a file
* on a folder

**The request concerns the support of 'Ajust namespaces' on:**
* a project
* the solution

Let's go ?

**Comment by pauer24** [comment]
Sure! I think it'd be a great improvement. Never found the time to
implement it.

Thanks mate!

P.

**Comment by Tr4ncer** [comment]
The development will take a little time but I have not given up.

A first part has been delivered in #25

**Comment by pauer24** [comment]
There is no hurry :)
